### DISCUSSION TITLE
failed install kubevela from helm

### DISCUSSION THREAD

**Discussion by zhuanxuhit** [kubevela/kubevela#2313] [Q&A]
运行 helm install --debug  --create-namespace -n vela-system kubevela kubevela/vela-core，后报错
环境是使用的 kind

Error: failed post-install: warning: Hook post-install vela-core/templates/addons-default/terraform.yaml failed: Internal error occurred: failed calling webhook "validating.core.oam-dev.v1beta1.initializers": Post "https://vela-core-webhook.vela-system.svc:443/validating-core-oam-dev-v1beta1-initializers?timeout=10s": dial tcp 10.96.242.96:443: connect: connection refused
helm.go:94: [debug] failed post-install: warning: Hook post-install vela-core/templates/addons-default/terraform.yaml failed: Internal error occurred: failed calling webhook "validating.core.oam-dev.v1beta1.initializers": Post "https://vela-core-webhook.vela-system.svc:443/validating-core-oam-dev-v1beta1-initializers?timeout=10s": dial tcp 10.96.242.96:443: connect: connection refused

**Comment by wonderflow** [comment]
应该是 vela的Controller还没启动起来导致自动安装插件失败，我们在 v1.1.4 小版本里将自动安装插件默认去掉了，改成安装完vela-core 以后再手动按需安装，可以试下 v1.1.4 哈
